addons/sprite-properties/userscript.js

```diff
@@ -17,20 +16,36 @@ export default async function ({ addon, global, console, msg }) {
       let spriteId = e.detail.action.editingTarget;
       if (!spriteId) return;
       let spriteIndex = e.detail.action.targets.findIndex((el) => el.id === spriteId);
-      injectInfoButton(spriteIndex);
+      // The focused sprite might not be in the target list if, for example, we are editing a clone.
+      if (spriteIndex !== -1) {
+        injectInfoButton(spriteIndex);
+      }
     }
   });
-  // Add a single event listener on the entire grid to take advantage of event bubbling
-  spriteGrid.addEventListener("click", (e) => {
-    let doubleClick = e.detail === 2;
-    if (doubleClick) togglePropertiesPanel();
+
+  // Open the properties panel when double clicking in the sprite grid
+  document.addEventListener("click", (e) => {
+    if (e.detail === 2 && e.target.closest('[class^="sprite-selector_scroll-wrapper_"]')) {
+      togglePropertiesPanel();
+    }
   });
+
   // Close properties panel when mouse leaves the entire sprite panel
-  spriteContainer.addEventListener("mouseleave", () => autoHidePanel());
+  document.body.addEventListener(
+    "mouseleave",
+    (e) => {
+      if (e.target.matches('[class*="sprite-selector_sprite-selector_2KgCX"]')) {
```

Comment:
Was this a mistake? Why use wildcard if we're going to specify the hash (2KgCX)?

Reply:
yeah